development/extensions/skeleton_extension.rst

```diff
@@ -0,0 +1,750 @@
+========================
+phpBB Skeleton Extension
+========================
+
+The Skeleton Extension is a tool for extension developers. Its 
+purpose is to kick start your extension development projects.
+
+What is a Skeleton? It's a working extension, based off the phpBB 
+Acme Demo extension. When you want to develop an extension, creating 
+a Skeleton will generate most of the initial files you would need,
+so you don't have to.
+
+This means you can jump straight into writing code for your extension,
+and bypass the mundane task of creating the initial files, methods
```

Comment:
I don't think most non native people have seen the word mundane before, I am sure there is a better word for it :)